public/components/integrations/components/setup_integration.tsx

```diff
@@ -415,6 +415,9 @@ export function SetupIntegrationForm({
               </EuiFormRow>
             </>
           ) : null}
+          {/* Bottom bar will overlap content if there isn't some space at the end */}
+          <EuiSpacer />
```

Comment:
nit. Euispacer has props for size="l"/"xl". May be use that.

Reply:
Tried xxl but it was still too small, 2 default spacers did the trick